packages/vanilla-components/src/components/referral-list/referral-list.tsx

```diff
@@ -3,46 +3,221 @@ import { css } from "emotion";
 import { uuid } from "../../utilities";
 import { API } from "../../services/WidgetHost";
 
+/**
+ * @uiName Referral List
+ */
 @Component({
   tag: "sqh-referral-list",
   styleUrl: "referral-list.scss",
 })
 export class ReferralList {
-  // general in dropdown
+  /**
+   * Hide the component
+   *
+   * @uiName Is Hidden
+   * @uiGroup general
```

Comment:
Props that are not meant to go in an according and are meant to be displayed at the top level will not need a uiGroup